tests/functional/gcs/test_collections.py

```diff
@@ -40,6 +40,13 @@ def test_get_collection_list_include_param(client):
     assert "include" in req.params
     assert req.params["include"] == "foo,bar"
 
+    client.get_collection_list(mapped_collection_id="MAPPED_COLLECTION")
+    assert get_last_request().params.get("mapped_collection_id") == "MAPPED_COLLECTION"
+
+    filters = ["mapped_collections", "created_by_me"]
+    client.get_collection_list(filter=filters)
+    assert get_last_request().params.get("filter") == ",".join(filters)
```

Comment:
```suggestion
    assert get_last_request().params.get("filter") == ",".join(filters)
    
    client.get_collection_list(filter="created_by_me")
    assert get_last_request().params.get("filter") == "created_by_me"
```

Let's add one more for the string case, yeah?

Reply:
Oops, yep. Good suggestion, thanks!

Reply:
Sorry that my suggestion had trailing whitespace! I didn't even realize that GitHub had done that.